pkg/registries/registries.go

```diff
@@ -248,14 +355,14 @@ func EditRegistriesConfig(config *sysregistriesv2.V2RegistriesConf, insecureScop
 			// So, if there is any mirror defined for a more specific sub-scope of mirrorSet.Source,
 			// it must already exist with non-empty reg.Mirrors.
 			if scope != mirroredScope && ScopeIsNestedInsideScope(scope, mirroredScope) && len(reg.Mirrors) == 0 {
-				reg.MirrorByDigestOnly = mirroredReg.MirrorByDigestOnly
 				updated, err := mirrorsAdjustedForNestedScope(mirroredScope, scope, mirroredReg.Mirrors)
 				if err != nil {
 					return err
 				}
 				reg.Mirrors = updated
 			}
 		}
+
```

Comment:
(Absolutely non-blocking: drop this line? I don’t think it separates “paragraphs” / individual ideas when placed here.)